api/v3/openapi.yaml

```diff
@@ -627,6 +627,45 @@ paths:
         '200':
           $ref: '#/components/responses/GetReplicaResponse'
 
+  /clusters/{cluster_id}/topics/{topic_name}/records:
+    parameters:
+      - $ref: '#/components/parameters/ClusterId'
+      - $ref: '#/components/parameters/TopicName'
+      - $ref: '#/components/parameters/PartitionId'
+
+    post:
+      summary: 'Produce records to the given topic.'
+      description: '<p>Produce records to the given topic, returning delivery reports for each
+                    record produced.</p>
+                    <p>This is a streaming API. To use it in streaming mode, you need to specify the
```

Comment:
nit: _This is a streaming API_ followed by _To use it in streaming mode_ sounds a bit confusing to me. If this is a streaming API, is there any other mode but streaming mode?

Reply:
I'm assuming that this can be used in non-streaming mode too...

if yes, perhaps we can word it as "This API can be used in streaming mode by setting "Transfer-Encoding: chunked" on the header".

Reply:
Done.

Reply:
🔥